### PR TITLE
📖 Add In-place upgrades feature group

### PR DESCRIPTION
**Pull request by g-gaston**
**What this PR does / why we need it**:
As proposed in office hours, creating a working group to work together in a proposal.

**Which issue(s) this PR fixes**:
Starting with https://github.com/kubernetes-sigs/cluster-api/issues/9489

/area documentation

### CODE REVIEW

**Comment by mayur-tolexo** [comment]
I would like to be a part of this.

**Comment by fabriziopandini** [comment]
/lgtm

Great to see this happening!
I personally think this is the right time for the project to look into this, looking forward to see this moving forward

**Comment by fabriziopandini** [comment]
/hold 
for the meeting time to be added + give others time to review

**Review by killianmuldoon — COMMENTED** [review]
/lgtm

Looking forward to seeing what comes from this work!

File: docs/community/20231016-in-place-upgrades.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Regular, summarized updates of group progress will be provided during weekly Cluster API office hours on Wednesdays @ 10:00 PT.
+
+Chat with stakeholders on Kubernetes [Slack](http://slack.k8s.io/) in the [cluster-api](https://kubernetes.slack.com/archives/C8TSNPY4T) channel.
+
+## Stakeholders
+
+Primary Stakeholders are listed below:
+
+- Dharmjit Singh (@dharmjit, VMware)
+- Guillermo Gaston (@g-gaston, AWS)
```
Comment (vincepri) [inline docs/community/20231016-in-place-upgrades.md:47]: Could you add @sdodson and myself to this list?

File: docs/community/20231016-in-place-upgrades.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Feature Group for Managed Kubernetes in Cluster API
```
Comment (dharmjit) [inline docs/community/20231016-in-place-upgrades.md:2]:
```suggestion
title: Feature Group for In-place Upgrades support in Cluster API
```

File: docs/community/20231016-in-place-upgrades.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+authors:
+  - "@g-gaston"
+reviewers:
+    - "@dharmjit"
+    - "@vincepri"
+    - "@sbueringer”"
```
Comment (dharmjit) [inline docs/community/20231016-in-place-upgrades.md:8]: typo
```suggestion
    - "@sbueringer"
```

File: docs/community/20231016-in-place-upgrades.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Regular, summarized updates of group progress will be provided during weekly Cluster API office hours on Wednesdays @ 10:00 PT.
+
+Chat with stakeholders on Kubernetes [Slack](http://slack.k8s.io/) in the [cluster-api](https://kubernetes.slack.com/archives/C8TSNPY4T) channel.
+
+## Stakeholders
+
+Primary Stakeholders are listed below:
+
+- Alexander Demicev (@@alexander-demicev, SUSE)
```
[the remark this answers is not in the document]
Comment (g-gaston) [reply docs/community/20231016-in-place-upgrades.md:46]: 🤦🏻

**Comment by g-gaston** [comment]
Meeting and google doc added, I think this is ready to go :)
/unhold

File: docs/community/20231016-in-place-upgrades.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+## Scope
+
+The scope of this effort will be the following:
+
+1. Define the scope of what In-place upgrades means in the context of Cluster API.
+2. Write a CAPI proposal with the recommended solution.
+
+## Communication
+
+We will meet on [Wednesdays at 08:00 PT (Pacific Time)][zoomMeeting]. [Convert to your timezone](https://dateful.com/time-zone-converter?t=09:00&tz=PT%20%28Pacific%20Time%29). Meeting notes will be documented in this [Google Doc](https://docs.google.com/document/d/1GmRd6MyQ0mWAoJV6rCHhZTSTtKMKHdJzhXm0BLBXOnw).
```
Comment (furkatgofurov7) [inline docs/community/20231016-in-place-upgrades.md:36]: I think you are missing to add `zoomMeeting` reference at the end.

File: docs/community/20231016-in-place-upgrades.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+## Stakeholders
+
+Primary Stakeholders are listed below:
+
+- Mayur Das (@mayur-tolexo, VMware)
+- Alexander Demicev (@alexander-demicev, SUSE)
+- Scott Dodson (@sdodson, Red Hat)
+- Guillermo Gaston (@g-gaston, AWS)
+- Furkat Gofurov (@furkatgofurov7, SUSE)
+- Dharmjit Singh (@dharmjit, VMware)
```
Comment (furkatgofurov7) [inline docs/community/20231016-in-place-upgrades.md:51]:
```suggestion
- Dharmjit Singh (@dharmjit, VMware)

[zoomMeeting]: https://zoom.us/j/861487554
```

File: docs/community/20231016-in-place-upgrades.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+The scope of this effort will be the following:
+
+1. Define the scope of what In-place upgrades means in the context of Cluster API.
+2. Write a CAPI proposal with the recommended solution.
+
+## Communication
+
+We will meet on [Wednesdays at 08:00 PT (Pacific Time)][zoomMeeting]. [Convert to your timezone](https://dateful.com/time-zone-converter?t=09:00&tz=PT%20%28Pacific%20Time%29). Meeting notes will be documented in this [Google Doc](https://docs.google.com/document/d/1GmRd6MyQ0mWAoJV6rCHhZTSTtKMKHdJzhXm0BLBXOnw).
+
+Regular, summarized updates of group progress will be provided during weekly Cluster API office hours on Wednesdays @ 10:00 PT.
```
Comment (furkatgofurov7) [inline docs/community/20231016-in-place-upgrades.md:38]:
```suggestion
Regular, summarized updates of group progress will be provided during weekly Cluster API office hours on Wednesdays @ 10:00 PT on [Zoom][zoomMeeting].
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
Few last nits we need to fix :)

**Review by furkatgofurov7 — COMMENTED** [review]
Thank you @g-gaston!

/lgtm

**Comment by Oats87** [comment]
I would love to also be a part of this!

**Comment by g-gaston** [comment]
> I would love to also be a part of this!

We'll be meeting Wednesdays at 8am PST in https://zoom.us/j/861487554
Pls join us!

**Comment by Oats87** [comment]
> > I would love to also be a part of this!
> 
> We'll be meeting Wednesdays at 8am PST in https://zoom.us/j/861487554 Pls join us!

Apologies for not joining. I was told "one hour before CAPI meeting" and had assumed it was at 9 AM PST rather than 8 AM PST.

I will do my best to join it next week!

**Comment by sbueringer** [comment]
This seems fine to me. Should we merge @vincepri @fabriziopandini ?

/lgtm

**Comment by pshail** [comment]
I also would love to be a part of this effort

**Review by fabriziopandini — COMMENTED** [review]
/lgtm
/approve

Awesome folks!

**Pull request merged**